migrations/000042_ReAddExportImporterTables.up.sql

```diff
@@ -0,0 +1,58 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+-- Top level configuration for other key server export files
+-- that should be imported into this server.
+CREATE TABLE ExportImport(
+    id SERIAL PRIMARY KEY,
+    index_file VARCHAR(500) NOT NULL,
+    export_root VARCHAR(500) NOT NULL,
+    region VARCHAR(5) NOT NULL,
+    from_timestamp TIMESTAMPTZ NOT NULL,
```

Comment:
We do do some searching on these timestamps and others, so I think we need an index.